src/gwt/acesupport/acemode/rmarkdown.js

```diff
@@ -193,6 +193,16 @@ oop.inherits(Mode, MarkdownMode);
          return this.$outdent.autoOutdent(session, row);
    };
 
+   this.getIndentForOpenBrace = function(openBracePos)
```

Comment:
This definition is necessary to ensure that the closing `}` is formatted correctly when inserting a newline as in e.g.

```
foo <- function() {|}
```

with `|` denoting cursor position.